src/views/v2/day/top-bar/today.php

```diff
@@ -11,14 +11,9 @@
  *
  * @version 4.9.3
```

Comment:
Not a stopper, but every time we update a template we need to set this number as TBD so it's replaced with the new version (and eventually when this is live, users can know which templates have been updated).

Reply:
Ok, thanks, will update.

Reply:
Thanks 👍  Not a stopper now, but something we should get used to do every time we touch a template, to create the habit.